src/d2q9_inc/conf.mk

```diff
@@ -0,0 +1,2 @@
+ADJOINT=0
+TEST=TRUE
```

Comment:
I think we can exclude this from tests - I'm planning to reduce the number of tested models (or make some different policy for testing) in near future.